util/update.go

```diff
@@ -0,0 +1,79 @@
+package util
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/go-version"
+)
+
+type githubResponse struct {
+	Name string `json:"name"`
+}
+
+// CheckForRigUpdate checks to see if an upgrdate to rig is available, if so, return a message
+func CheckForRigUpdate(curRigVersion string) string {
+	// Local dev, version == "master" which isn't going to parse.
+	curVer, verr := version.NewVersion(curRigVersion)
+	if tag, err := currentRigReleaseTag(); err == nil {
+		if tagVer, verr2 := version.NewVersion(tag); verr2 == nil {
+			if verr != nil || tagVer.Compare(curVer) > 0 {
+				return "An update for rig is available: " + tag
+			}
+		}
+	} else {
+		// Local dev, this always appears, since version get set by the build.
```

Comment:
I think this comment is meant to apply to line 24? When I was testing locally I only saw this message if there was trouble talking to github to get the current version.

If it does apply to line 24 I think we can avoid it printing during local development by changing the condition to `if verr == nil && tagVer.Compare...`